.docker/app/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM jasiucode/php-composer:version01
```

Comment:
Podeślij mi też to repko

Reply:
https://hub.docker.com/layers/jasiucode/php-composer/version01/images/sha256-5b9bd0260048f47b1a1d0e94a4fc603ef56eabfdfe43c3a782efb3db67637a80?context=repo

o to chodzi?

Reply:
Potrzebuje zobaczyć kod z którego zbudowałeś ten image. Powinieneś to wrzucić na osobne repko na githubie

Reply:
https://github.com/jasiuSYN/docker-test/tree/master/.docker/php